DEMO-101.txt

```diff
@@ -8,4 +8,6 @@ Who's using Flutter?
 2.ebay
 3.Square
 4.Alibaba Group
-
+5. BMW
+6. CapitalOne
+7.Square
```

Comment:
The company "Square" name repeated. Please check and commit again for the review.

Reply:
Done